manifests/service.pp

```diff
@@ -29,11 +34,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-class openvpn::service {
-  service { 'openvpn':
-    ensure     => running,
-    enable     => true,
-    hasrestart => true,
-    hasstatus  => true,
+class openvpn::service ($manage_service = true) {
+
+  if $manage_service {
```

Comment:
Instead of adding a parameter, please reference `$::openvpn::manage_service` directly (as you do for the systemd part.